### PR TITLE
refactoringOpeningHoursTests

### PR DESCRIPTION
**Pull request by OrleneMitchell**

### CODE REVIEW

File: core/src/test/java/nav/statusplattform/core/repositories/OpeningHoursRepositoryTest.java [hunk 1]
```diff
@@ -388,17 +421,134 @@ void removeOpeningHoursFromService() {
...
+                .setName(getRandomFromArray(areaNames))
+                .setRule(getRandomFromArray(rules));
+    }
+
+    private String getRandomFromArray(ArrayList<String> array) {
+        //Hit skal man ikke komme
+        if (array.isEmpty()) return null;
+        Random random = new Random();
+        return array.get(random.nextInt(array.size()));
+    }
```
Comment (x10an14-nav) [inline core/src/test/java/nav/statusplattform/core/repositories/OpeningHoursRepositoryTest.java:449]: I suggest you keep code like this gathered in one common location. I don't consider this code special, at least wrt to openinghours.

**Review by x10an14-nav — CHANGES_REQUESTED** [review]
As discussed in person =)

1. Consolidate code more. Many near identical pieces of code between SampleData & OpeninghoursRepositoryTest.
1. Make sure you can argue the case why, if total #of code don't go down.
1. Make PR metadata (title. etc) reflect scope of contained changes.

File: core/src/test/java/nav/statusplattform/core/repositories/OpeningHoursRepositoryTest.java (hunk 1 above)
Comment (OrleneMitchell) [reply core/src/test/java/nav/statusplattform/core/repositories/OpeningHoursRepositoryTest.java]: Thanks for the tips Christian. I have only made changes to this code before your message. Its now apart of SampleData.

**Comment by OrleneMitchell** [comment]
[like]  Mitchell, Orlene reacted to your message:
________________________________

**Review by x10an14-nav — COMMENTED** [review]
LGTM! Just want to double-check all tests pass on my machine once I get a chance, then I'll give a thumbs up and you can merge =)

**Review by x10an14-nav — APPROVED** [review]
Got a chance to do it immediately after all, good work!

**Pull request merged**